build/centos6-puppet4/Gemfile.lock

```diff
@@ -1,133 +0,0 @@
-GEM
-  remote: https://rubygems.org/
-  specs:
```

Comment:
Any background behind removing `Gemfile.lock` files?
How that can affect CI builds/reproducibility?

Reply:
Both the Gemfile and Gemfile.lock's are no longer specific to each distribution (thanks to a standardized Ruby version in Puppet 4/5)

Instead the Gemfile from the repo root is used to defined gems for unit testing.